apps/cowswap-frontend/src/modules/tenderly/utils/generateSimulationData.ts

```diff
@@ -18,16 +18,40 @@ export function generateSimulationDataToError(
   )
 }
 
+function convertBalanceDiffToLowerCaseKeys(data: BalancesDiff): BalancesDiff {
+  return Object.entries(data).reduce((acc, [outerKey, innerObj]) => {
+    const lowerOuterKey = outerKey.toLowerCase()
```

Comment:
why inner/outer? what are these keys? Is it user account and token address? why not stating that in the name?

Reply:
Changed the names of the variables for a better documentation!